### PR TITLE
Add kubeconfig path

### PR DESCRIPTION
**Pull request by fengshunli**
This PR fixes #

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?

add kubeconfig parameter, you can switch clusters

### CODE REVIEW

File: pkg/kube/resources.go [hunk 1]
```diff
@@ -242,12 +255,28 @@ func CreateResourceProviderFromYaml(yamlContent string) *ResourceProvider {
...
+			logrus.Errorf("Error fetching KubeConfig: %v", err)
+			return nil, err
+		}
+	} else {
+		kubeConf, err = config.GetConfig()
+		if err != nil {
+			logrus.Errorf("Error fetching KubeConfig: %v", err)
+			return nil, err
+		}
+	}
```
Comment (rbren) [inline pkg/kube/resources.go:279]: Do you think we can extract all this logic out into a helper function? That way it's not duplicated here and `CreateResourceProviderFromResource`. (Also it looks like we missed the `KubeContext` option there!)

**Review by rbren — COMMENTED** [review]
Thanks for the contribution! This looks like a very helpful change.

FWIW I _think_ you can get the same functionality today by setting the `KUBECONFIG` env var.

File: pkg/kube/resources.go (hunk 1 above)
Comment (fengshunli) [reply pkg/kube/resources.go:279]: KubeContext is specified by specifying kubeconfig, without kubeconfig, KubeContext is meaningless

**Comment by rbren** [comment]
Hmm--not quite sure what you mean. How would `KUBECONFIG=~/kconf polaris audit` differ from `polaris audit --kubeconfig=~/kconf`?

You might be right, just want to make sure I understand

**Comment by fengshunli** [comment]
Right now I'm using --kubeconfig
image

**Comment by fengshunli** [comment]
> Hmm--not quite sure what you mean. How would `KUBECONFIG=~/kconf polaris audit` differ from `polaris audit --kubeconfig=~/kconf`?
> 
> You might be right, just want to make sure I understand

Do you mean that I need to change --kubeconfig to KUBECONFIG @rbren

**Comment by rbren** [comment]
No, no need to change anything in this PR. I was just saying that running
```
export KUBECONFIG=./kconf
polaris audit
```
or more simply
```
KUBECONFIG=./kconf polaris audit
```
are both supported today. I think that achieves the same thing if you wanted a workaround.

I like having the `--kubeconfig` flag too though!

File: pkg/kube/resources.go (hunk 1 above)
Comment (rbren) [reply pkg/kube/resources.go]: Right exactly. But we're not using `KubeContext` in `CreateResourceProviderFromResource`, even though we have a kubeconfig there.

**Comment by fengshunli** [comment]
> KUBECONFIG

KUBECONFIG is also available for the method

**Pull request closed without merging**